services/movie_session.py

```diff
@@ -0,0 +1,46 @@
+from django.utils.dateparse import parse_date
+from db.models import Movie, CinemaHall, MovieSession
+
+
+def create_movie_session(movie_show_time: int |float,
+                         movie_id: id,
+                         cinema_hall_id: int) -> Movie | CinemaHall:
```

Comment:
The return type hint is incorrect. The function should return a `MovieSession` object, not `Movie | CinemaHall`.